internal/server/store_test.go

```diff
@@ -1798,12 +1798,57 @@ var _ = ginkgo.Describe("Scoped storage functions", func() {
 		txn.DatasetEntities["people"] = entities
 		txn.DatasetEntities["places"] = entities1
 
+		// get counts before txn
+		dsInfo, err := store.NamespaceManager.GetDatasetNamespaceInfo()
+
+		// check counts after txn
+		datasets := []string{"core.Dataset"}
+		dsEntity, err := store.GetEntity(fmt.Sprintf("%s:%s", dsInfo.DatasetPrefix, "people"), datasets)
+		Expect(err).To(BeNil())
+
+		if dsEntity != nil {
+			items, ok := dsEntity.Properties[dsInfo.ItemsKey]
+			Expect(ok).To(BeTrue())
+			Expect(items).To(Equal(float64(5)))
```

Comment:
Nothing wrong here, just wanted to mention `Expect(items).To( BeEquivalent(5) )` as alternative syntax when you dont really care about the type.